api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -534,7 +534,22 @@ public async Task<IActionResult> UploadFile([FromRoute] long id, [FromQuery] lon
         else await _fileReferenceService.UploadCleanUpAsync(new ContentFileReference(content, files.First()), _storageOptions.GetUploadPath());
 
         if (_workOrderHelper.ShouldAutoTranscribe(content.Id)) await _workOrderHelper.RequestTranscriptionAsync(content.Id);
-
+        var fileReference = content.FileReferences.First();
+        if (fileReference.ContentType.StartsWith("video/", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
It won't ever be null because `.First()` will throw an error if it's null already.  The real question is whether `.First()` is always going to return the correct reference.  We don't currently allow more than one file, but it might not be wise to always assume.